### PR TITLE
Fix navbar overlap on Getting Started page (refactored)

### LINKED ISSUE #2640: Getting started videos - YouTube player interface bug
What you did (step by step)
-tried to advance video in embedded getting started videos 

Where does this happen on the OSF?
-https://osf.io/getting-started

What you expected
-to be able to advance the video

What actually happened
-control bar would disappear as the cursor neared it.  Tested this with Michael as well and he experienced the same issue.  

Potential causes
-maybe related to an update that Youtube made in the player

Suggest a solution

What will it look like when this issue is resolved?
-user will be able to advance video from within the embedded video player

### PR DESCRIPTION
**Pull request by zkraime**
<h2>Purpose:</h2> 
<p>There was an issue (#2641) regarding the navigation bar on the Getting Started page in that it overlaps the text/video if you scroll down or resize the window making it too small.</p>
<p>This PR prevents the overlap.</p>
<p> Also closes #2640</p>

<h2>Changes:</h2>
<p>The navbar is now affixed to the top <i>and</i> left, and there's a buffer between itself and the text/videos so when you reduce the screen size by enough, the navbar disappears before it overlaps the text/videos.</p>
<b>before:</b>
image
<b>after:</b>
screen shot 2015-05-21 at 10 49 02 am
<b>Resize</b>
screen shot 2015-05-21 at 10 58 56 am

### CODE REVIEW

**Comment by GaryKriebel** [comment]
@zik3qe .... does this PR replace https://github.com/CenterForOpenScience/osf.io/pull/2782?

**Comment by zkraime** [comment]
@GaryKriebel Yes! I indicated as such to @sloria. 
The problem was that the first PR was comparing my develop with osf develop when I should have been comparing my fix branch. 
So, after I refactored the code, instead of merging with develop and pushing like I did before, I just created a new PR using my menu_overlap_fix branch.

**Comment by GaryKriebel** [comment]
Thanks, @zkraime ... I will close the other.

**Comment by zkraime** [comment]
@GaryKriebel Isn't it already closed? 
Either way, thanks

**Comment by GaryKriebel** [comment]
It is... I did not see that you closed it prior to my reply.

**Comment by samanehsan** [comment]
Would it be possible to reduce the gap between the menu and the getting started text? Now that the menu is affixed to the far left, it looks like the text is too far to the right: 
screen shot 2015-06-02 at 11 39 35 am

**Comment by brianjgeiger** [comment]
This seems like the wrong solution. We should be taking advantage of the bootstrap columns in this case, rather than working around them. The menu should take up the first column or two, and the content can take the rest of the columns. If the window shrinks past a certain point, you should use the bootstrap responsive capabilities to make the menu appear on the top and the content below that.  (See https://osf.io/settings/ for an example of this)

**Comment by zkraime** [comment]
@brianjgeiger Thanks, I'll change it to reflect these characteristics

**Comment by brianjgeiger** [comment]
:+1:

**Comment by zkraime** [comment]
@brianjgeiger 
After a bit of a bootstrap/css learning curve, I got the content to wrap beneath the navbar but the issue is that the navbar resizes itself as you scroll down (to display the many nested links) such that it's seemingly impossible to display the proper chunk of content while on its respective nested link. 
Is there a way to force the vertical size to be static while hiding the overflow of nested links in a scroll? 

On a different note, @PatrickEGorman sent a PR for issue #2640. 
He said he made edits to the layout which will most likely conflict with changes that I were to push. 
Should I wait for that PR to be closed first, @GaryKriebel?

**Comment by lbanner** [comment]
That would be the conservative approach @zkraime -- or you could pull his branch and make your changes there, and resubmit as a combined PR. Let me know if I am off on this, or better way, @brianjgeiger

**Comment by brianjgeiger** [comment]
Handling merge conflicts is a valuable learning opportunity.

For handling the vertical nature, Maybe take a look at how they manage it on http://getbootstrap.com/components/

**Comment by PatrickEGorman** [comment]
I'm pretty sure I already solved the menu problem when I was working on the youtube issue.  @zkraime feel free and double check it if you want

File: website/templates/public/pages/help/components.mako [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
-    <h4>Components</h4>
-    <p>Components are like sub-projects that help you organize different parts of your research. Components have their own privacy settings, contributors, wiki, add-ons, and files.</p>
-    <div class="col-md-8 col-md-offset-2 col-sm-12">
-        <div class="gs-video embed-responsive embed-responsive-16by9">
-            <div class="embed-responsive-item youtube-loader" id="kotF8Pnurpw"></div>
-        </div>
+<span id="components" class="anchor"></span>
+<h4>Components</h4>
+<p>Components are like sub-projects that help you organize different parts of your research. Components have their own privacy settings, contributors, wiki, add-ons, and files.</p>
+<div class="col-md-8 col-md-offset-2 col-sm-12">
```
Comment (samanehsan) [inline website/templates/public/pages/help/components.mako]: This div should be inside of a `<div class="row">` element

**Comment by samanehsan** [comment]
Can you pull from osf develop again?

**Comment by samanehsan** [comment]
I updated the PR description to include that it closes #2640 now that the commits from Patrick's PR have been merged in.

**Comment by samanehsan** [comment]
Pass complete @zkraime! Please ping me when you're done making those changes :tada:

**Comment by samanehsan** [comment]
The dataverse paragraph is slightly over-indented. Can you make it line up with the other add-on descriptions? Thanks! 
screen shot 2015-06-11 at 1 28 56 pm

**Comment by zkraime** [comment]
Absolutely!

**Comment by samanehsan** [comment]
@sloria, this is ready for another pass! :sunny:

**Comment by sloria** [comment]
Looks good. :octopus:

**Pull request merged**